test/validators/format.js

```diff
@@ -47,13 +47,34 @@ describe("format validation", () => {
       })
 
 
+      it("allows RegExp flag pattern", function() {
+
+        //zipcode regex
+        var pattern = new RegExp('^[0-9]{8}$')
+         
+        const samples = [
+           "05541030",37130000
+        ]
+ 
+        for (const value of samples) {
+            // given
+            const validations = { format: pattern }
+            // when
+            const ret = validate(value, validations)
+            // then
+            assert.deepStrictEqual(ret, { value: value, errors: [] })
+        }
+ 
+       })
+
+
     it("allows work with multiple validations", function() {
 
         //zipcode regex
         var pattern = /^[0-9]{8}$/
          
         const samples = [
-           "05541030"
+            "05541030",37130000
```

Comment:
I don't think the SUT (system under test) here is the values per se but `multiple validations`. given thar I don't see the need for multiple values

Reply:
Don't you think it could happen that a field is mandatory (or not) and still needs a regex if it is filled out?

Reply:
Sure. But here you are testing `presence: true`, so this test tests for mandatory field. And the description/intention of you test is clear: `multiple validations`, which I assume it is just to make sure the `format` validators does not conflict with others validators. So having multiple values doesn't change the SUT.